java/piranha/src/test/java/com/uber/piranha/EnumConstantTest.java

```diff
@@ -225,4 +683,169 @@ public void testRemoveEnumFieldValueMatchTest() {
             "}")
         .doTest();
   }
+
+  /** Test to cleanup an enum with only one remaining enum constant */
+  @Test
+  public void testRemoveEnumWithSingleConstant() {
```

Comment:
As a note: In a way, this is contained in [this](https://github.com/uber/piranha/blob/master/java/piranha/src/test/java/com/uber/piranha/CorePiranhaTest.java#L123) test, BUT I am happy to include this case because: a) it makes it explicit that we are testing for this behavior, b) if we ever refactor `positiveRemoveImport` to mention other flags, it would remove the safety net for this case, while this test's sole purpose is catching that :)